src/docs/frontend/using-styled-components.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: Using Styled Components
+---
+
+To apply styles to HTML we use a css-in-js library called [Emotion](https://emotion.sh/). This allows us to define styles in the same file where the component is defined and used. Styles are scoped to avoid global selectors and having to manually specify classnames at strings. You can also leverage the theme object to reference common values and utilities.
+
+The best styles are the ones you don’t write - whenever possible use existing components.
+
+## Basics
+
+To use Style Components pull in `styled` and apply your css. For consistent spacing you should also use `space()`:
+
+```tsx
+import styled from '@emotion/styled';
+import {space} from 'sentry/styles/space';
+
+const Numeric = styled('span')`
+  font-variant-numeric: tabular-nums;
+`;
+
+const SideBySide = styled('div')`
+  display: flex;
+  gap: ${space(2)};
+  flex-wrap: wrap;
+  align-items: flex-start;
+`;
```

Comment:
👍🏻 maybe worth mentioning that even styled components deserve good names. e.g., `SideBySide` is a great name, something like `Container` or `StyledPanel` are less good

Reply:
I'm gonna followup on this. I don't really know what convention we should push.. there's lots of bad examples too, but i know whatever we write will be optimistic/idealistic for the future.